packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -12544,6 +12544,150 @@ void main() {
       );
     });
   });
+
+  group('Spell check', () {
+    testWidgets(
+        'Spell check configured properly when spell check disabled',
+            (WidgetTester tester) async {
+          await tester.pumpWidget(
+            MaterialApp(
+              home: EditableText(
+                controller: TextEditingController(text: 'A'),
+                focusNode: FocusNode(),
+                style: const TextStyle(),
+                cursorColor: Colors.blue,
+                backgroundCursorColor: Colors.grey,
+                cursorOpacityAnimates: true,
+                autofillHints: null,
+              ),
+            ),
+          );
+
+          final EditableTextState state =
+            tester.state<EditableTextState>(find.byType(EditableText));
+          expect(state.spellCheckEnabled, isFalse);
+        });
```

Comment:
Nit: I think usually the closing braces here should align with the `testWidgets` above.  Here and in the other tests.